taxi/forms.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class CleanLicenseNumberMixin:
+    def clean_license_number(self):
```

Comment:
you can simplify this by using MaxLengthValidator and RegexValidator on license_number field in a form